packages/nimble-components/src/rich-text-viewer/styles.ts

```diff
@@ -1,12 +1,58 @@
 import { css } from '@microsoft/fast-element';
 import { display } from '@microsoft/fast-foundation';
-import { bodyFont, bodyFontColor } from '../theme-provider/design-tokens';
+import {
+    bodyFont,
+    bodyFontColor,
+    bodyFontLineHeight,
+    linkActiveFontColor,
+    linkFontColor,
+    smallPadding
+} from '../theme-provider/design-tokens';
 
 export const styles = css`
     ${display('flex')}
 
     :host {
         font: ${bodyFont};
+        outline: none;
         color: ${bodyFontColor};
+        width: auto;
+        overflow: auto;
+        height: 100%;
+        min-height: 36px;
```

Comment:
Consistency on [logical properties](https://web.dev/learn/design/internationalization/#logical-properties) for different writing modes of languages (especially right to left like Arabic,etc.)

```suggestion
        inline-size: auto;
        overflow: auto;
        block-size: 100%;
        min-block-size: 36px;
```

Reply:
Updated